protected/humhub/modules/admin/resources/js/humhub.admin.PendingRegistrations.js

```diff
@@ -0,0 +1,65 @@
+humhub.module('admin.PendingRegistrations', function(module, require, $) {
+    var object = require('util').object;
+    var Widget = require('ui.widget').Widget;
+    var client = require('client');
+    var urlDeleteSelected;
+
+    var PendingRegistrations = function(node, options) {
+        Widget.call(this, node, options);
+    };
+
+    object.inherits(PendingRegistrations, Widget);
+
+    PendingRegistrations.deleteAllSelected = function(evt) {
+        client.post(evt).then(function() {
+            var keys = $("#grid").yiiGridView("getSelectedRows");
+            $.ajax({
+                url: urlDeleteSelected,
+                type: "POST",
+                data: {id: keys},
+            })
+        }).catch(function(e) {
+            module.log.error(e, true);
+        })
+    };
+
+    PendingRegistrations.deleteAll = function(evt) {
+        client.post(evt).then(function() {
+            that.reload();
+        }).catch(function(e) {
+            module.log.error(e, true);
+        })
+    };
+
+    function hasChecked($checkBoxes) {
+        var result = false;
+        $checkBoxes.each(function () {
+            if ($(this).prop("checked")) {
+                result = true;
+                return false;
+            }
+        });
+        return result;
+    }
+
+    PendingRegistrations.prototype.init = function() {
+        urlDeleteSelected = this.options.urlDeleteSelected;
```

Comment:
I'd rather directly check `this.options.urlDeleteSelected` in the `deleteAllSelected ` instead of adding this static variable.

Reply:
Done!